_docs/tasks/security/faq.md

```diff
@@ -43,9 +43,22 @@ type: markdown
 
   If Istio Auth is enabled, http and tcp health check from kubelet will not
   work since they do not have Istio Auth issued certs. A workaround is to
-  use command option for health check, e.g., one can install curl in the
-  service pod and curl itself within the pod. The Istio team is actively
-  working on a solution.
+  use a [liveness command](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-command)
+  for health check, e.g., one can install curl in the service pod and curl itself
+  within the pod. The Istio team is actively working on a solution.
+
+  An example of readinessProbe:
+
+  ```
+  livenessProbe:
+    exec:
+      command:
+      - curl
+      - -f
+      - http://localhost:{$PORT}
```

Comment:
the {$PORT} is strange (that’s meant as an example/ for user to change to the actual port right?)
what about : 
```
   - http://localhost:8080/health # replace with your actual port and health endpoint
```